userguide/sharing.rst

```diff
@@ -1061,88 +1060,80 @@ provides more details for each configurable option.
-   +-------------------------+--------------+-------------+-----------------------------------------------------------------------------------------------------+
+   +-------------------------+--------------+-------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Setting                 | Value        | Advanced    | Description                                                                                                                                       |
+   |                         |              | Mode        |                                                                                                                                                   |
+   |                         |              |             |                                                                                                                                                   |
+   +=========================+==============+=============+===================================================================================================================================================+
+   | Path                    | browse       |             | Select the volume, dataset, or directory to share.                                                                                                |
+   |                         | button       |             |                                                                                                                                                   |
+   |                         |              |             |                                                                                                                                                   |
+   +-------------------------+--------------+-------------+---------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Use as home share       | checkbox     |             | Set to allow this share to hold user home directories. Only one share can be the home share. Note that lower case names for user home directories |
+   |                         |              |             | are strongly recommended, as Samba maps usernames to all lower case. For example, the username John will be mapped to a home directory named      |
+   |                         |              |             | :file:`john`. If the :guilabel:`Path` to the home share includes an upper case username, delete the existing user and recreate it in              |
+   |                         |              |             | :menuselection:`Accounts --> Users` with an all lower case :guilabel:`Username`. Return to :menuselection:`Sharing --> SMB` to create the home    |
```

Comment:
Perhaps put an internal :ref: to the Accounts --> Users section of the guide? Just a thought.